### DISCUSSION TITLE
What happened to class prefix's in v5?

### DISCUSSION THREAD

**Discussion by dclstn** [rsuite/rsuite#2110] [Q&A] [answered]
v4 rsuite
webpack.config.js
```
new webpack.DefinePlugin({
      __RSUITE_CLASSNAME_PREFIX__: JSON.stringify('random-'),
}),
```
outputs classnames as such: `<div class="random-modal"/>`,
v5 no longer supports this and returns the base prefix, can we expect this functionality to return?

**Comment by SevenOutman** [comment, marked as answer]
Hi @dclstn , we removed this custom global prefix feature in v5 because it makes reading and maintaining the styles really hard after we adopted CSS custom properties. There's not yet a certain conclusion whether or when we'll add this feature back. I'd assume it's not coming back during v5 lifetime as far as I consider.
